external_search.py

```diff
@@ -1652,12 +1652,16 @@ def __init__(self, collections=None, media=None, languages=None,
         These minor arguments were made into unnamed keyword arguments to
         avoid cluttering the method signature:
 
-        `param excluded_audiobook_data_sources`: A list of DataSources that
+        :param excluded_audiobook_data_sources: A list of DataSources that
         provide audiobooks known to be unsupported on this system.
         Such audiobooks will always be excluded from results.
 
-        `param allow_holds`: If this is False, books with no available
+        :param allow_holds: If this is False, books with no available
         copies will be excluded from results.
+
+        :param updated_after: If this is set to a datetime, only books
+        whose Work records (~bibliographic metadata) have been updated since
+        that time will be included in results.
```

Comment:
I'm a little worried about adding more and more tiny changes to the `Filter` object and having custom `Facets` object program it, rather than giving the `Facets` object the ability to directly modify the underlying Elasticsearch filter. This is definitely easier to test, though, and it keeps almost all the Elasticsearch code in `external_search.py`.